lib/register_client.rb

```diff
@@ -129,8 +128,8 @@ def validate_register_integrity(rsf, current_entry_number)
       end
     end
 
-    def download_rsf(register, phase, start_entry_number)
-      RestClient.get("https://#{register}.#{phase}.openregister.org/download-rsf/#{start_entry_number}")
+    def download_rsf(start_entry_number)
+      RestClient.get(@register_url + "/download-rsf/#{start_entry_number}")
```

Comment:
I think it's cleaner to read if we use a single way to operate a string:

```ruby
RestClient.get("#{@register_url}/download-rsf/#{start_entry_number}")
```

Besides that, are we ensuring the `@register_url` does not contain a trailing `/`? A safer way to approach this would be using the `URI` module https://docs.ruby-lang.org/en/2.1.0/URI.html:

```ruby
URI.parse(@register_url).merge("download-rsf").merge(start_entry_number).to_s
``` 

(the `.to_s` bit is because I'm assuming `RestClient.get` is expecting a string, would be much cleaner if it expected a URI I suppose.

Reply:
@arnau I've addressed this in my latest commit.